README.md

```diff
@@ -1 +1,90 @@
-# lantern-back-end
\ No newline at end of file
+# FHIR Target Querier
+A service to send http requests to get capability statements from FHIR endpoints
+
+## Building And Running
+
+The Endpoint Querier takes one arguement, a JSON file containing the endpoints which the service should query. The list of endpoints provided in `./resources/EndpointSources.json` was taken from https://fhirendpoints.github.io/data.json.
+
+```bash
+go install ./...
```

Comment:
This was working for me as-is because I had my GOPATH set to the repository directory -- but this only worked because of the src/  directory in the repository. Having a src/ directory in the repository isn't recommended so I am moving the endpoints and fhir packages outside of the src directory. This will also make the imports appear more correct eg: "github.com/onc-healthit/lantern-back-end/endpoints/fetcher" instead of "github.com/onc-healthit/lantern-back-end/src/endpoints/fetcher"